samples/tcp-echo/src/Dockerfile

```diff
@@ -23,12 +23,16 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o tcp-echo main.go
 ARG BASE_DISTRIBUTION=default
 
 # The following section is used as base image if BASE_DISTRIBUTION=default
+# No tag available https://hub.docker.com/_/scratch?tab=description
+# hadolint ignore=DL3006
 FROM scratch as default
 
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
-FROM gcr.io/distroless/static as distroless
+# hadolint ignore=DL3007
+FROM gcr.io/distroless/static:latest as distroless
```

Comment:
please fix latest if it can be fixed.

Reply:
I'm afraid it can only be fixed by something cringy like 8bef63d2c8654ff89358430c7df5778162ab6027 or  c9115ffeeeae0a89ba0c9531b27fdc661add9445 or  de7f1306895f22d4a4d2fab59d723178c7121937 or f5916ee5446549a8bfdc471502605ddf4ecaff79
It's sadly non-human readable.